tools/check_format.py

```diff
@@ -674,6 +686,9 @@ def checkSourcePath(file_path):
                                                 "Java proto option 'java_outer_classname' not set")
       error_messages += errorIfNoSubstringFound("\n" + PROTO_OPTION_JAVA_MULTIPLE_FILES, file_path,
                                                 "Java proto option 'java_multiple_files' not set")
+    with open(file_path) as f:
+      if PROTO_OPTION_GO_PACKAGE in f.read():
```

Comment:
Awesome. Can you add some `check_format` tests for these conditions?

Reply:
Done.